learn/advanced/chronological_filtering_sorting.md

```diff
@@ -0,0 +1,116 @@
+# Filtering and sorting by date
+
+In this guide, you will learn about Meilisearch's approach to date and time values, how to prepare your dataset for indexing, and how to chronologically sort and filter search results.
+
+## Preparing your documents
+
+To filter and sort search results based on chronological values, your documents must have a numeric field containing a [UNIX timestamp](https://kb.narrative.io/what-is-unix-time). In this videogames dataset, the release year is formatted as a timestamp:
+
+```json
+[
+  {
+    "id": 0,
+    "title": "Return of the Obra Dinn",
+    "genre": "adventure",
+    "release_date": 1514761200
+  },
+  {
+    "id": 1,
+    "title": "The Excavation of Hob's Barrow",
+    "genre": "adventure",
+    "release_date": 1640991600
+  },
+  {
+    "id": 2,
+    "title": "Bayonetta 2",
+    "genre": "action",
+    "release_date": 1388530800
+  }
+]
```

Comment:
After sticking these dates into a unix timestamp converter, I was a bit confused that they didn't match up with the actual release dates of the games in question. Then I realized that they are supposed to be the release year, not the release date. I think it would make more sense for this guide to use specific dates as examples, since filtering or sorting by year is easy even without using a timestamp.